System/Win32/FileMapping.hsc

```diff
@@ -1,4 +1,4 @@
-{-# LANGUAGE Trustworthy #-}
```

Comment:
This change is causing older GHCs to fail
```
System/Win32/FileMapping.hsc:25:1:
    Foreign: Can't be safely imported! The module itself isn't safe.
```

Reply:
Fixed.